hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/CreateDagVisitor.java

```diff
@@ -117,6 +120,22 @@ public Vertex onFilter(FilterPhysicalRel rel) {
         return vertex;
     }
 
+    public Vertex onSort(SortPhysicalRel rel) {
+        Vertex vertex = dag.newUniqueVertex("Sort",
+                ProcessorMetaSupplier.forceTotalParallelismOne(
+                        ProcessorSupplier.of(
+                                sortP(ExpressionUtil.comparisonFn(getCollations(rel))
+                                )
+                        ),
+                        localMemberAddress
+                )
+        );
+
+        connectInput(rel.getInput(), vertex, edge -> edge.distributeTo(localMemberAddress).allToOne(""));
```

Comment:
Maybe we could/should do a 2-stage sort (something like in `SortTransform`)?

Reply:
We absolutely should.